### ISSUE TITLE
Suggestion!

### ISSUE THREAD

**Issue by thanone2** [DeckThemes/CSSLoader-Desktop#32]
Can you add support for Audio Loader in the windows app? Or maybe could someone tell me how to manually install the audio .zip files?

**Comment by exzelis** [comment]
Agreed, I hope they will add Audio Loader too in CSS Loader Desktop since Valve keep removing our favorite custom audio files.

**Comment by Pikmingamer9539** [comment]
i also agree
